### PR TITLE
boa: add more boa examples

### PR DESCRIPTION
**Pull request by yorkie**

### CODE REVIEW

File: example/boa/argparse/test [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+const boa = require('@pipcook/boa');
+const { int } = boa.builtins();
+const argparse = boa.import('argparse');
+const sys = boa.import('sys');
+
+const commander = argparse.ArgumentParser();
+commander.add_argument('--foo', boa.kwargs({ action: 'store_const', 'const': 42 }));
+commander.add_argument('--num', boa.kwargs({
+  type: int,
```
Comment (Txiaozhe) [inline example/boa/argparse/test:13]: Extra comma

File: example/boa/tf2/word-embedding.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+const { layers } = keras;
+
+const embedding_layer = layers.Embedding(1000, 5);
+const [
+  [train_data, test_data],
+  info
+] = tfds.load('imdb_reviews/subwords8k', boa.kwargs({
+  split: tuple([tfds.Split.TRAIN, tfds.Split.TEST]),
+  with_info: true,
+  as_supervised: true,
```
Comment (Txiaozhe) [inline example/boa/tf2/word-embedding.js:21]: Extra comma

File: example/boa/tf2/word-embedding.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                                .padded_batch(10, boa.kwargs({ padded_shapes }));
+const test_batches = test_data.shuffle(1000)
+                              .padded_batch(10, boa.kwargs({ padded_shapes }));
+
+const embedding_dim = 16;
+const model = keras.Sequential([
+  layers.Embedding(encoder.vocab_size, embedding_dim),
+  layers.GlobalAveragePooling1D(),
+  layers.Dense(16, boa.kwargs({ activation: 'relu' })),
+  layers.Dense(1, boa.kwargs({ activation: 'sigmoid' })),
```
Comment (Txiaozhe) [inline example/boa/tf2/word-embedding.js:38]: Extra comma

File: example/boa/tf2/word-embedding.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+model.compile(boa.kwargs({
+  optimizer: 'adam',
+  loss: 'binary_crossentropy',
+  metrics: ['accuracy']
+}));
+
+const result = model.fit(train_batches, boa.kwargs({
+  epochs: 1,
+  validation_data: test_batches,
+  validation_steps: 20,
```
Comment (Txiaozhe) [inline example/boa/tf2/word-embedding.js:51]: Extra comma

File: example/boa/tf2/word-embedding.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+}));
+
+{
+  const vecs = fs.createWriteStream('vecs.tsv');
+  const meta = fs.createWriteStream('meta.tsv');
+  const weights = model.layers[0].get_weights()[0]
```
Comment (Txiaozhe) [inline example/boa/tf2/word-embedding.js:57]: missing ';'

File: example/boa/argparse/test [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+'use strict'
+
+const boa = require('@pipcook/boa');
+const { int } = boa.builtins();
+const argparse = boa.import('argparse');
+const sys = boa.import('sys');
+
+const commander = argparse.ArgumentParser();
+commander.add_argument('--foo', boa.kwargs({ action: 'store_const', 'const': 42 }));
```
Comment (Txiaozhe) [inline example/boa/argparse/test:11]: remove ''

Comment (yorkie) [reply example/boa/argparse/test:11]: const is the javascript keyword, quoted might be more meaningful.

File: example/boa/argparse/test (hunk 1 above)
Comment (yorkie) [reply example/boa/argparse/test:13]: The project rule allows this, and you could point out the reason.

File: example/boa/tf2/word-embedding.js (hunk 2 above)
Comment (yorkie) [reply example/boa/tf2/word-embedding.js:21]: Ditto

**Comment by yorkie** [comment]
Removed the example from lint ignores, thank you @Txiaozhe.

**Pull request merged**